subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -656,19 +656,33 @@ Note that abstract classes are not executed. Gradle also scans up the inheritanc
 
 If you don't want to use test class detection, you can disable it by setting `scanForTestClasses` to false. This will make the test task only use includes / excludes to find test classes. If `scanForTestClasses` is false and no include / exclude patterns are specified, the defaults are ???`\**/*Tests.class`???, ???`*\*/*Test.class`??? and ???`*\*/Abstract*.class`??? for include and exclude, respectively.
 
+[NOTE]
+====
+When using http://junit.org/junit5/docs/current/user-guide[JUnit Platform], we simply pass all test classes to `JUnit Platform` and let it decide what should be run. In this case, `scanForTestClasses` has no effect.
+====
+
 [[test_grouping]]
 ==== Test grouping
 
-JUnit and TestNG allows sophisticated groupings of test methods.
+JUnit/JUnit Platform and TestNG allows sophisticated groupings of test methods.
```

Comment:
"allow" instead of "allows" ?

Reply:
Nice catch! It's a typo existing for a long time.